### PR TITLE
fix(cli): can select bun in PackageManagers

### LINKED ISSUE #176: In `cli` page, To select `bun` in `PackageManagers` has error.
### Provide the exact quote of the error or issue

In `cli` page, To select `bun` in `PackageManagers` has error.
```
text: Application error: a client-side exception has occurred (see the browser console for more information).
console: TypeError: Cannot read properties of undefined (reading 'type')
```

### Does the docs page already exist? Please link to it.

https://yamada-ui.com/styled-system/cli

### Are you willing to participate in fixing this issue and create a pull request with the fix?

Yes

### PR DESCRIPTION
**Pull request by komura-c**
Closes #176

## Description

> In cli page, can select bun in PackageManagers

## Current behavior (updates)

> In cli page, To select bun in PackageManagers has error. 

## New behavior

> In cli page, can select bun in PackageManagers

## Is this a breaking change (Yes/No):

No

## Additional Information

`bun install` add package to `devDependencies` by default
https://bun.sh/docs/cli/install

### CODE REVIEW

File: contents/styled-system/cli.ja.mdx [hunk 1]
```diff
@@ -17,6 +17,7 @@ CLIは、カスタマイズしたテーマを読み取り、`node_modules/@yamad
     pnpm: "pnpm add --dev @yamada-ui/cli",
     npm: "npm install --save-dev @yamada-ui/cli",
     yarn: "yarn add --dev @yamada-ui/cli",
+    bun: "bun install @yamada-ui/cli",
```
Comment (KenyaMasuko) [inline contents/styled-system/cli.ja.mdx:20]:
```suggestion
    bun: "bun add --dev @yamada-ui/cli",
```
I referenced the following [this document](https://bun.sh/docs/cli/add), and I was wondering if this code is appropriate for installing the package with bun. What do you think?🤔

**Review by KenyaMasuko — COMMENTED** [review]
Please confirm the point I mentioned!

File: contents/styled-system/cli.ja.mdx (hunk 1 above)
Comment (komura-c) [reply contents/styled-system/cli.ja.mdx]: Thanks for the review! You are correct and I fixed it.

**Review by KenyaMasuko — APPROVED** [review]
Thanks👏

**Pull request merged**